src/consts.rs

```diff
@@ -92,10 +84,8 @@ impl PartialEq for Constant {
         match (self, other) {
             (&Constant::Str(ref ls, ref lsty), &Constant::Str(ref rs, ref rsty)) => ls == rs && lsty == rsty,
             (&Constant::Binary(ref l), &Constant::Binary(ref r)) => l == r,
-            (&Constant::Byte(l), &Constant::Byte(r)) => l == r,
```

Comment:
Bytes have been removed. The are changed to `Constant::Int(ConstInt::U8(...))`. I'm assuming there's no reason to keep bytes around.